DEPS.xwalk

```diff
@@ -17,7 +17,7 @@
 # Edit these when rolling DEPS.xwalk.
 # -----------------------------------
 
-chromium_crosswalk_rev = '4281fabd3a7b4b5b87f5653e51a851045bbd5d55'
+chromium_crosswalk_rev = '507c2f4ad33694d5ec2a8549c5dbeb97b150a596'
```

Comment:
This hash only available when the chromium PR get merged. Let's wait for that one.